src/concepts/proxying/invisible.md

```diff
@@ -4,28 +4,28 @@
 Invisible proxying is disabled by default, [view the guide on how to enable it](/guides/invisible_proxying).
 :::
 
-Generally applications can be configured to use a forward proxy to connect to their target.
-For example, we use the browsers proxy configurations to setup Caido.
-This is what allows us to intercept the traffic.
+Generally, applications can be configured to use a forward proxy to connect to their targets.
+
+This is the case with Caido. By manually configuring the connection settings (_or by using a browser extension such as [FoxyProxy](https://getfoxyproxy.org/)_), we are able to proxy the traffic the browser generates, intended for a web server, through Caido.
 
 <img alt="Firefox proxying setup" src="/_images/proxying_firefox.png" center width=600px style="filter: brightness(85%);" />
 
-When proxying "regularly", the application will first send a `CONNECT` request to establish a tunnel before sending the request itself:
+When proxying in this "normal" context, the application will send a `CONNECT` request to establish a TCP tunnel before sending the request itself:
 
 ```http
 CONNECT https://example.com/ HTTP/1.1
 Host: localhost:8080
 ```
 
-When an application cannot be configured to use a proxy (_considered a "thick client"_), we need an alternative method.
-This is where invisible proxying is interesting.
+However, some applications are considered to be "thick clients", and cannot be configured to use a proxy. In such cases, "invisible proxying" (_aka "transparent proxying"_) becomes an alternative method.
+
+In this mode, Caido acts as the destination server, making the application unaware that it is communicating through a proxy. As a result, the application sends HTTP requests directly to Caido, without first sending a `CONNECT` request. This behavior is similar to how reverse proxies like Nginx&trade; operate.
+
+This means that Caido needs to listen on the same port as the destination server (_typically `443` or `80`_).
 
-In this mode, Caido acts as the destination server and the application is not aware that it is talking to a proxy.
-It will thus send the HTTP requests directly without a `CONNECT`.
-This is similar to how reverse proxies like Nginx:tm: work.
+Additionally, DNS Rewrite rules are required to override the local DNS resolution of the target domain, pointing it `127.0.0.1` (_the listening address of Caido_). Then, another rule needs to revert the resolution back to the real IP address of the server to ensure proper forwarding.
```

Comment:
This is unclear. You only need one DNS rewrite rule in Caido to revert to the real IP. But you also need to add a DNS entry on your computer to force the domain to 127.0.0.1